core/types/bid.go

```diff
@@ -193,5 +193,7 @@ type MevParams struct {
 	ValidatorCommission   uint64 // 100 means 1%
 	BidSimulationLeftOver time.Duration
 	GasCeil               uint64
+	GasPrice              *big.Int // Minimum avg gas price for bid block
```

Comment:
why add "GasPrice" and "Version", seems they are not used right now?

Reply:
For builders' information, builders could decide how to interact with each validator through this information